ArraysHomework/ArraysHomework.cs

```diff
@@ -55,6 +59,8 @@ public void Arrays()
 
             // Swap the last and first value of this array.
             string[] swapMyValues = { "first", "middle", "last" };
+            swapMyValues[0] = "last";
```

Comment:
How would you do this if you didn't know the length of the array or what the values were? If, for example, the values came from user input rather than being hard-coded in the code?

Reply:
swapMyValues[0] = swapMyValues[SwapMyValues.Length - 1]